Core/include/Acts/Seeding/InternalSpacePoint.hpp

```diff
@@ -27,7 +29,9 @@ class InternalSpacePoint {
   InternalSpacePoint(std::size_t index, const SpacePoint& sp,
                      const Acts::Vector3& globalPos,
                      const Acts::Vector2& offsetXY,
-                     const Acts::Vector2& variance);
+                     const Acts::Vector2& variance,
+                     const std::optional<float>& globalTime,
```

Comment:
Not sure the optional cover const ref is with it over just passing the optimal by value here.